lib/assessment.md

```diff
@@ -2,53 +2,71 @@ This file is a **markdown** file. Markdown is a special way of formatting text,
 
 In responding to some of these questions, you may be asked to write HTML or CSS code. Please do so using the following Markdown formatting.
 
-##### For HTML
-    ```HTML
-    <html>
-    ...
-    </html>
-    ```
-
-##### For CSS
-    ```CSS
-    ul {
-      ...
-    }
-    ```
+# For HTML
 
-Using this format will make it easier for us to grade you, so that we can get everyone their feedback faster. Thanks for your help!
+````
+```HTML
+<html>
+...
+</html>
+```
+````
 
-## Question 1
+# For CSS
 
-When we say that the web is a service, what does that mean? Explain the interaction in terms of clients and servers.
-<!-- your answer starts here -->
+````
+```CSS
+ul {
+  ...
+}
+```
+````
 
-<!-- your answer ends here -->
+Using this format will make it easier for us to grade you, so that we can get everyone their feedback faster. Thanks for your help!
+
+## Question 1
+When we say that the web is a service, what does that mean? Explain the interaction in terms of clients and servers. <!-- your answer starts here --> The web is a service in that it represents a consistent framework for displaying documents across the internet <!-- your answer ends here -->
```

Comment:
This explanation isn't really complete - how do clients and servers fit into this scheme?